addons/mail/static/src/discuss/voice_message/common/voice_recorder.xml

```diff
@@ -9,7 +9,7 @@
                 'mw-0': !state.recording,
                 'text-danger': state.limitWarning,
             }" style="font-variant-numeric: tabular-nums;"><span class="d-flex text-truncate" t-esc="state.elapsed"/></div>
-            <button t-att-disabled="state.isActionPending or props.composer.voiceAttachment" class="border-0 btn rounded-pill p-1" t-att-title="title" t-on-click="onClick"><i t-att-class="!state.recording ? 'fa fa-fw fa-microphone' : 'fa fa-fw fa-circle text-danger o-mail-VoiceRecorder-dot'"/></button>
+            <button t-att-disabled="state.isActionPending or props.composer.voiceAttachment" class="border-0 btn rounded-pill p-1" t-att-class="{ 'fs-3': ui.isSmall }" t-att-title="title" t-on-click="onClick"><i t-att-class="{ 'fa fa-fw fa-circle text-danger o-mail-VoiceRecorder-dot': state.recording, 'fa fa-fw fa-microphone': !state.recording, 'fa-lg': ui.isSmall }"/></button>
```

Comment:
`btn btn-link btn-danger` instead of `text-danger`?

Reply:
Again, `.btn-link.btn-danger` doesn't combine as you think:
<img width="915" alt="Screenshot 2024-06-24 at 15 51 15" src="https://github.com/odoo/odoo/assets/6569390/09c0c0e7-80ee-405c-acc7-240c97d0291d">

`.btn-link` has a pretty strange stylistic choice. IMO we should consider it only it its default behavior, otherwise this is counter-intuitive and probably not worth using it.